### PR TITLE
Update Android sdkmanager for all platforms

### PR DESCRIPTION
**Pull request by yjbanov**
The new version will gives us the `--licenses` option in the `sdkmanager` command.

### CODE REVIEW

File: dev/bots/README.md [hunk 1]
```diff
@@ -119,6 +119,11 @@ Instructions to update the Android Tools version that the bots download by execu
 3. Run upload_android_tools.py -t sdk
    `$ dev/bots/upload_android_tools.py -t sdk`
 
+4. Run `dev/bots/android_tools/sdk/tools/bin/sdkmanager --update`. On Windows, run `sdkmanager.bat` instead. If the
```
Comment (goderbauer) [inline dev/bots/README.md]: Can you also update the versions mentioned a few lines above?

**Review by goderbauer — APPROVED** [review]
LGTM

File: dev/bots/README.md (hunk 1 above)
Comment (yjbanov) [reply dev/bots/README.md]: Can I delete them instead? They seem redundant as it's easy to get the current versions by running `sdkmanager --list` and it's (apparently) easy to forget to update this list here.

Comment (goderbauer) [reply dev/bots/README.md]: Yeah, that is good as well. Maybe you can document how to get the latest version from sdkmanager instead?

Comment (yjbanov) [reply dev/bots/README.md]: Done.

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**